### DISCUSSION TITLE
How to Install Ganache on Ubuntu?

### DISCUSSION THREAD

**Discussion by 0xRadiant** [ConsenSys-archive/ganache#3189] [Q&A]
Have been running into issues with installing Ganache since it's provided as an AppImage. Has anyone given this a try successfully? Below is what I tried, and the current errors I'm seeing.

```bash
parallels@ubuntu-linux-20-04-desktop:~/Downloads$ chmod a+x ganache-2.5.4-linux-x86_64.AppImage 
parallels@ubuntu-linux-20-04-desktop:~/Downloads$ sudo ./ganache-2.5.4-linux-x86_64.AppImage 
./ganache-2.5.4-linux-x86_64.AppImage: 1: �: not found
./ganache-2.5.4-linux-x86_64.AppImage: 2: }�P7pys�H��_f��h: not found
./ganache-2.5.4-linux-x86_64.AppImage: 1: �b�@@: not found
./ganache-2.5.4-linux-x86_64.AppImage: 1: ELFAI: not found
./ganache-2.5.4-linux-x86_64.AppImage: 19: Syntax error: Unterminated quoted string
```

I have tried following some of the relatively straightforward suggestions such as https://askubuntu.com/questions/774490/what-is-an-appimage-how-do-i-install-it#774520. I could also just switch to using macOS, but curious to understand what it would take to get this working on Ubuntu.

**Comment by apdan510** [comment]
Radiant,

Can you share some more details about your machine with us? What CPU, what kind of RAM, exact Linux distro?

Thank you!
Alex | Truffle Support
